pkg/operator/http.go

```diff
@@ -209,105 +177,25 @@ func checkForFields(fields []string, vals url.Values) error {
-	if err != nil {
-		logger.WithError(err).Errorf("error getting report: %v", err)
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "error getting report: %v", err)
-		return
-	}
-
-	// Get the presto table to get actual columns in table
-	prestoTable, err := srv.prestoTableLister.PrestoTables(report.Namespace).Get(reportingutil.PrestoTableResourceNameFromKind("scheduledreport", report.Name))
-	if err != nil {
-		if k8serrors.IsNotFound(err) {
-			writeErrorResponse(logger, w, r, http.StatusAccepted, "ScheduledReport is not processed yet")
-			return
-		}
-		logger.WithError(err).Errorf("error getting presto table: %v", err)
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "error getting presto table: %v", err)
-		return
-	}
-
-	tableColumns := prestoTable.Status.Parameters.Columns
-	queryPrestoColumns, err := reportingutil.GeneratePrestoColumns(reportQuery)
-	if err != nil {
-		logger.WithError(err).Errorf("error converting ReportGenerationQuery columns to presto columns: %v", err)
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "error converting columns: %v", err)
-		return
-	}
-
-	prestoColumns, err := reportingutil.HiveColumnsToPrestoColumns(tableColumns)
-	if err != nil {
-		logger.WithError(err).Errorf("error converting PrestoTable hive columns to presto columns: %v", err)
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "error converting columns: %v", err)
-		return
-	}
-
-	if !reflect.DeepEqual(queryPrestoColumns, prestoColumns) {
-		logger.Warnf("report columns and table columns don't match, ReportGenerationQuery was likely updated after the report ran")
-		logger.Debugf("mismatched columns, PrestoTable columns: %v, ReportGenerationQuery columns: %v", prestoColumns, queryPrestoColumns)
-	}
-
-	tableName := reportingutil.ScheduledReportTableName(name)
-	results, err := srv.reportResultsGetter.GetReportResults(tableName, prestoColumns)
-	if err != nil {
-		logger.WithError(err).Errorf("failed to perform presto query")
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "failed to perform presto query (see operator logs for more details): %v", err)
-		return
-	}
-
-	if len(results) > 0 && len(prestoTable.Status.Parameters.Columns) != len(results[0]) {
-		logger.Errorf("report results schema doesn't match expected schema, got %d columns, expected %d", len(results[0]), len(prestoTable.Status.Parameters.Columns))
-		writeErrorResponse(logger, w, r, http.StatusInternalServerError, "report results schema doesn't match expected schema")
-		return
-	}
-
-	writeResultsResponseV1(logger, format, reportQuery.Name, reportQuery.Spec.Columns, results, w, r)
-}
 func (srv *server) getReport(logger log.FieldLogger, name, format string, useNewFormat bool, full bool, w http.ResponseWriter, r *http.Request) {
-	// Get the current report to make sure it's in a finished state
+	// Get the report to make sure it's isn't failed
```

Comment:
it isn't finished

Reply:
It should actually be `it isn't failed`, but yeah, the `it's` was wrong.